Demo/Program.cs

```diff
@@ -576,6 +577,13 @@ private static void Demo13()
             wb2.Save();                                                                    // Save the workbook
         }
 
+        private static void Demo14()

```

Comment:
Please remove the demo. The feature is a little bit too obscure to designate it as a single demo.
Testing of the feature will be taken into account as a one or some testcase(s) in the new major version which is currently in the dev branch under development